src/gui/MainWindow.cc

```diff
@@ -3208,23 +3262,9 @@ void MainWindow::showEditor()
 
 void MainWindow::hideEditor()
 {
-  auto e = (ScintillaEditor *) this->activeEditor;
-  if (windowActionHideEditor->isChecked()) {
-    // Workaround manually disabling interactions with editor by setting it
-    // to read-only when not being shown.  This is an upstream bug from Qt
-    // (tracking ticket: https://bugreports.qt.io/browse/QTBUG-82939) and
-    // may eventually get resolved at which point this bit and the stuff in
-    // the else should be removed. Currently known to affect 5.14.1 and 5.15.0
-    e->qsci->setReadOnly(true);
-    e->setupAutoComplete(true);
-    editorDock->close();
-  } else {
-    e->qsci->setReadOnly(false);
-    e->setupAutoComplete(false);
-    editorDock->show();
-  }
```

Comment:
I moved the fix into TabManager.
(but maybe the fix is not needed anymore...)

Reply:
It might be needed for the affected Qt versions, so we should probably conditionally enable the fix. It's not clear to me exactly where it's needed though, so finding the right condition could be less than trivial..

Reply:
We try to support roughly the time frame of still supported Ubuntu and/or Debian old-stable. For now that's 20.04LTS with 5.12.8, next step is 22.04LTS with 5.15.3. So even when dropping 20.04 support, the workaround might still be needed unfortunately.

Reply:
I reverted that change so it should be baseline code.